src/pages/workspace/companyCards/WorkspaceCompanyCardsPage.tsx

```diff
@@ -32,6 +33,7 @@ function WorkspaceCompanyCardPage({route}: WorkspaceCompanyCardPageProps) {
                 shouldShowOfflineIndicatorInWideScreen
             >
                 <View style={[styles.mt3, shouldUseNarrowLayout ? styles.workspaceSectionMobile : styles.workspaceSection]} />
+                <WorkspaceCompanyCardPageEmptyState route={route} />
```

Comment:
Can we consider rendering this based on condition or will it be handled in the next PR's?

Reply:
I think we'll be handling this in the next related PR's 👍